packages/locales/src/locales/en.json

```diff
@@ -33,6 +33,7 @@
   "conjunction.or_3+_1": ", ",
   "conjunction.or_3+_2": ", or ",
   "datafix.error.command-replaceitem": "/replaceitem was removed in 20w46a (the second snapshot of 1.17) in favour of /item",
+  "duplicate-components": "Duplicate component definitions for %0%",
```

Comment:
Rename the key to `mcfunction.parser.item.duplicate-components` and put it with the other `mcfunction` error messages in the file.